src/rt_redbug.erl

```diff
@@ -26,21 +26,22 @@
 -module(rt_redbug).
 
 -export([is_tracing_applied/0]).
--export([new/0]).
 -export([set_tracing_applied/1]).
 -export([stop/1]).
 -export([trace/2, trace/3]).
 
-%% Create a new ets to store globally whether we're tracing or not.
-new() -> 
-    ok.
-
 set_tracing_applied(TracingApplied) when is_boolean(TracingApplied) ->
-    lager:info("Setting apply tracing to ~p", [TracingApplied]),
-    rt_config:set(apply_traces, TracingApplied).
+    %% If this flag is set on the command line, but not in the config file
+    %% then switch it on.
+    Enabled = case rt_config:get(apply_traces, undefined) of
+        undefined -> TracingApplied;
+        ConfigValue -> ConfigValue
+    end,
+    lager:info("Setting apply tracing to ~p", [Enabled]),
```

Comment:
Would like to see this message be more explicit and higher level (`warning`?) given the potential impact on performance if tracing is enabled without the user realizing it.

Perhaps `Will enable any redbug traces contained in the test` (and `Will not enable any redbug traces contained in the test`) for the true/false messages.